data/outfits/god.txt

```diff
@@ -1,36 +1,86 @@
 outfit "Ruin-The-Fun God Mode"
 	"display name" "RTF God Mode"
 	category `RTF God Outfits`
-	cost 0
 	thumbnail "outfit/quarg quantum shield generator"
-	"mass" 0
 	hull 1000000
 	"hull repair rate" 1000000
-	"hull energy" 0
 	shields 1000000
 	"shield generation" 1000000
-	"cooling" 1000000
-	"ion resistance" 1000000
+	"high shield permeability" 0
+	"low shield permeability" 0
 	"disruption resistance" 1000000
+	"ion resistance" 1000000
+	"scramble resistance" 1000000
+	"slowing resistance" 1000000
 	"discharge resistance" 1000000
+	"corrosion resistance" 1000000
+	"leak resistance" 1000000
+	"burn resistance" 1000000
+	"piercing resistance" 1000000
+	"disruption protection" 1000000
+	"energy protection" 1000000
+	"force protection" 1000000
+	"fuel protection" 1000000
+	"heat protection" 1000000
+	"hull protection" 1000000
+	"ion protection" 1000000
+	"scramble protection" 1000000
 	"piercing protection" 1000000
+	"shield protection" 1000000
 	"slowing protection" 1000000
 	"discharge protection" 1000000
 	"corrosion protection" 1000000
 	"leak protection" 1000000
 	"burn protection" 1000000
-	bunks 1000000
-	description "Absolutely illegal."
+	description "Protect your ship against almost anything."
+
+outfit "Ruin-The-Fun God Scanner"
+	"display name" "RTF God Scanner"
+	category "RTF God Outfits"
+	thumbnail "outfit/outfit scanner"
+	"tactical scan power" 100000000
+	"asteroid scan power" 100000000
+	"outfit scan power" 100000000
+	"cargo scan power"  100000000
+	"outfit scan efficiency" 1000000
+	"cargo scan efficiency" 1000000
+	description "Scans far and wide."
 
+outfit "Ruin-The-Fun God Jammer"
+	"display name" "RTF God Jammer"
+	category "RTF God Outfits"
+	thumbnail "outfit/small radar jammer"
+	"scan concealment" 1000000
+	"scan interference" 1000000
+	"radar jamming" 1000000
+	"optical jamming" 1000000
+	description "Stops others from peeking."
```

Comment:
Attributes `scan concealment` and `scan interference` prevents scanning.
Attributes `radar jamming` and `optical jamming` interfere with missile tracking.

Is it intentional to have both in the same outfit?
Missiles using infrared tracking will not be affected by this outfit (I do not think there is any countermeasure other than cooling against this).

As for scan countermeasures, I didn't test if those high numbers are already enough to block scans completely, I'm just notting those attributes here:
- `cargo scan opacity`: Increases scan time.
- `outfit scan opacity`: Increases scan time.
- `inscrutable`: Prevents scanning outfits completely, BUT maybe that only affect the player, the scan would complete, but says you didnt find anything.

Reply:
These functions appears thematic to me so I put these together. I am open to splitting them.

Reply:
For jamming, I plan to have an outfit that, as opposed to `God Mode`, would be also capable of defending friendlies. It would be a super-powerful turret but that have jamming abilities.

But this is not exactly the same thing so...